core/src/main/java/jenkins/monitor/JavaVersionRecommendationAdminMonitor.java

```diff
@@ -78,7 +78,16 @@ public class JavaVersionRecommendationAdminMonitor extends AdministrativeMonitor
 
     static {
         NavigableMap<Integer, LocalDate> supportedVersions = new TreeMap<>();
-        supportedVersions.put(11, LocalDate.of(2024, 9, 30)); // Temurin: 2024-10-31
+        // Adjust Java 11 end of life date for weekly and LTS
+        if (Jenkins.VERSION.split("[.]").length > 2) {
+            // LTS will require Java 17 or newer beginning 30 Oct 2024
+            // https://groups.google.com/g/jenkinsci-dev/c/gsXAqOQQEPc/m/VT9IBYdmAQAJ
+            supportedVersions.put(11, LocalDate.of(2024, 10, 30)); // Temurin: 2024-10-31
```

Comment:
Given some remaining uncertainty around LTS dates it seems unnecessary to commit to another month.